test/integration/integration_suite_test.go

```diff
@@ -728,7 +728,7 @@ func assertVirtualMachineState(machine *clusterv1.Machine, vm *vmoprv1.VirtualMa
 	Expect(machine.Spec.Version).ShouldNot(BeNil(), "Error accessing nil Spec.Version for machine %s", machine.Name)
 	Expect(vm.Spec.VmMetadata).NotTo(BeNil())
 	// TODO: Aarti: not sure where to get this varaible from
-	//Expect(vm.Spec.VmMetadata.Transport).To(Equal(vmoprv1.VirtualMachineMetadataExtraConfigTransport))
+	//Expect(vm.Spec.VmMetadata.Transport).To(Equal(vmoprv1.VirtualMachineMetadataCloudInitTransport))
```

Comment:
we should uncomment this `Expect` call and remove the `TODO` at the same time

Reply:
Ack.